src/Diagnostics.DataProviders/KustoSDKClient.cs

```diff
@@ -184,9 +188,12 @@ public async Task<DataTable> ExecuteQueryAsync(string query, string cluster, str
                             exceptionType,
                             exceptionDetails
                             );
-
+                    if (attemptException is not null && totalResponseTime.TotalSeconds <= (double) _config.OverridableExceptions.Single(x => x[0].ToString().ToLower() == attemptException.Message.ToLower())[1] && IsSpecialRetryException(attemptException))
```

Comment:
Do you want to use contains here instead of comparing the exact exception message?